cdap-docs/application-templates/source/etl/templates/transformations/validator.rst

```diff
@@ -26,16 +26,15 @@ valid.
 Example: ``"validators": "core"``
 
 **validationScript:** Javascript that implements the function ``isValid``, taking a JSON object
-representation of the input record, and returning a ``Map<String, String>`` representing the result.
-The map should have these fields::
+representation of the input record, and returning a JSON representing the result.
+The result JSON should have these fields, errorCode and errorMsg can be ignored for valid records::
```

Comment:
The returned JSON will include these fields;  ``errorCode`` and ``errorMsg`` can be ignored for valid records::